modules/installation-registry-storage-block-recreate-rollout.adoc

```diff
@@ -14,6 +14,11 @@
 
 To allow the image registry to use block storage types such as vSphere Virtual Machine Disk (VMDK) during upgrades as a cluster administrator, you can use the `Recreate` rollout strategy.
 
+[IMPORTANT]
+====
+Block storage volumes are not recommended for use with image registry on production clusters. An installation where the registry is configured on block storage is not highly available because the registry cannot have more than 1 replica.
```

Comment:
Should 1 be spelled out? In the procedure it's written as `1` too (I think I'm to blame for this).

Reply:
Was on the fence cuz I was trying to match the procedure, where it makes sense to represent it as a value. But to answer your question, I think it does look better spelled out, so I've updated.